contracts/vendor/manifest.json

```diff
@@ -117,6 +207,142 @@
           "path": "contracts/{}"
         }
       }
+    },
+    "Timelock.sol": {
+      "source": {
+        "git": {
+          "repo": "git@github.com:compound-finance/compound-protocol.git",
+          "commit": "a3214f67b73310d547e00fc578e8355911c9d376",
+          "path": "contracts/Timelock.sol"
+        }
+      },
+      "patches": [
+        {
+          "oldStart": 1,
+          "oldLines": 29,
+          "newStart": 1,
+          "newLines": 34,
+          "lines": [
+            "-// SPDX-License-Identifier: BUSL-1.1",
+            "-pragma solidity 0.8.15;",
+            "+// SPDX-License-Identifier: BSD-3-Clause",
```

Comment:
Do we actually want to change the license for the Timelock?

Reply:
good question. i updated it to match our other Comet contracts, but I don't have a strong opinion on this

Reply:
Might be a question for 👨‍🍳 🍯

Reply:
Probably fine, not sure how much it matters and to who